plugins/git-prompt/gitstatus.py

```diff
@@ -27,6 +28,18 @@ def get_tagname_or_hash():
         return hash_
     return None
 
+# Re-use method from https://github.com/magicmonty/bash-git-prompt to get stashs count
+def get_stash():
+    cmd = Popen(['git', 'rev-parse', '--git-dir'], stdout=PIPE, stderr=PIPE)
+    so, se = cmd.communicate()
+    stash_file = '%s%s' % (so.decode('utf-8').rstrip(), '/logs/refs/stash')
```

Comment:
You cannot remove `decode('utf-8')` below and add it here
If decoding below breaks the output, maybe re-encoding it before printing is the right solution.